samples/config-apollo-sample/src/main/resources/microservice.yaml

```diff
@@ -26,7 +26,7 @@ apollo:
     serverUri: http://127.0.0.1:8070
     serviceName: apollo-test
     env: DEV
-    clusters: test-cluster
+    clusters: default
```

Comment:
Please add some explanation about this setting.

Reply:
Done